apps/client/src/app/components/user-account-access/user-account-access.component.ts

```diff
@@ -115,6 +115,8 @@ export class GfUserAccountAccessComponent implements OnDestroy, OnInit {
       .subscribe((params) => {
         if (params['createDialog']) {
           this.openCreateAccessDialog();
+        } else if (params['editDialog']) {
+          this.openUpdateAccessDialog(params['editDialog']);
```

Comment:
`params['editDialog']` is a boolean, not an id! (cf. [tags](https://github.com/ghostfolio/ghostfolio/blob/main/apps/client/src/app/components/admin-tag/admin-tag.component.ts#L75))

Reply:
Ups